### PR TITLE
[stable/instana-agent] Add Kubernetes cluster name configuration parameter

### PR DESCRIPTION
**Pull request by wiggzz**
#### What this PR does / why we need it:

Adds support for the `INSTANA_KUBERNETES_CLUSTER_NAME` environment variable as a helm chart value.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/instana-agent/README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ The following table lists the configurable parameters of the Instana chart and t
 | `rbac.create`                      | Whether RBAC resources should be created                                | `true`                                       |
 | `serviceAccount.create`            | Whether a ServiceAccount should be created                              | `true`                                       |
 | `serviceAccount.name`              | Name of the ServiceAccount to use                                       | `instana-agent`                              |
+| `kubernetes.clusterName`           | Display name of the monitored Kubernetes Cluster (overrides zone.name)  | `nil`                                        |
```
Comment (mdonkers) [inline stable/instana-agent/README.md]: I'm a bit divided by using `kubernetes` for naming the variable, as HELM charts should work with OpenShift as well. Until now, we only used "kubernetes" for documentation, nothing that cannot easily be changed. Unfortunately can't directly provide a better alternative though...
Perhaps any ideas from others?

Comment (wiggzz) [reply stable/instana-agent/README.md]: Yea I thought about this. I guess Kubernetes is the upstream project for all the other ones which is why I went with it. We could make it more generic like `cluster.name`?

Comment (dlbock) [reply stable/instana-agent/README.md]: Yeah, I like `cluster.name`.

Should we also add an explanation of the differences between `zone.name` and this new configuration? I'm not sure if people could get confused by which one they need to set?

Comment (mdonkers) [reply stable/instana-agent/README.md]: `cluster.name` sounds good to me as well.

**Comment by wiggzz** [comment]
Closing this in favor of a different approach that doesn't require additional configuration.

**Comment by wiggzz** [comment]
I've reopened this to allow configuration via the environment variable.

I've made it now only necessary to specify `zone.name` or `cluster.name`.

**Comment by nfisher** [comment]
Might've missed it but what happens when the cluster.name is set but zone.name isn't?

**Comment by wiggzz** [comment]
> Might've missed it but what happens when the cluster.name is set but zone.name isn't?

The zone will be pulled from different sensors which detect the location of the host based on the cloud  provider (e.g. availability zone on AWS)

**Comment by dlbock** [comment]
Going to be interesting to see if this generates or solves more customer tickets.

/lgtm

**Pull request merged**